### ISSUE TITLE
Route cache generation not possible due Closure

### ISSUE THREAD

**Issue by forxer** [freekmurze/laravel-glide#26]
The following code should just use a Controller:

```
$this->app['router']->get($glideConfig['baseURL'].'/{all}', function () use ($glideConfig)
```

Because cache generation may fail now because of laravel-glide :

```
php artisan route:cache
Route cache cleared!

  [LogicException]
  Unable to prepare route [img/{all}] for serialization. Uses Closure.
```

**Comment by freekmurze** [comment]
Hi, this is already fixed by pull request #20 

Make sure you are using version 2.2.0 or above.

**Comment by forxer** [comment]
oh yes sorry, I was using an outdated version, thank you
